daisy_tutorials/modify_image/emacs_image.wf.json

```diff
@@ -0,0 +1,57 @@
+{
+    "Name": "emacs-image",
+    "Sources": {
+        "emacs_install.sh": "./emacs_install.sh"
+    },
+    "Steps": {
+        "create-disks": {
+            "CreateDisks": [
+                {
+                    "Name": "instance-disk",
```

Comment:
To be consistent with the styleguide, name the disk: "disk-install"

Reply:
Fixed.